vs-alimpsim error

When building it fails because of vs-alimpsim 
```
CMake Error at CMakeLists.txt:30 (add_subdirectory):
  add_subdirectory given source "module/vs-alimpsim" which is not an existing
  directory.
```
You have to delete this line in the cmake list
https://github.com/silagokth/vesyla-suite-4/blob/079d30ecb5aba8beed40319fc7777678fa373e8e/CMakeLists.txt#L30

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,7 +27,6 @@ set(INSTALL_SHARED_OUTPUT_DIRECTORY share/${PROJECT_NAME})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/modules/")
 
 add_subdirectory("module/vs-manas")
-add_subdirectory("module/vs-alimpsim")
 add_subdirectory("module/vs-component")
 add_subdirectory("module/vs-testcase")
 add_subdirectory("module/vs-schedule")

